pages/common/pueue-completions.md

```diff
@@ -0,0 +1,16 @@
+# pueue completions
+
+> Generates shell completion files for bash, elvish, fish, power-shell, zsh.
```

Comment:
```suggestion
> Generates shell completion files for bash, elvish, fish, powershell, zsh.
```

Reply:
@she3o please apply this.  `power-shell` with the dash is technically incorrect.